src/test/java/core/basesyntax/strategy/TransationProcessorImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.impl.BalanceOperationHandler;
+import core.basesyntax.service.operation.impl.PurchaseOperationHandler;
+import core.basesyntax.service.operation.impl.ReturnOperationHandler;
+import core.basesyntax.service.operation.impl.SupplyOperationHandler;
+import core.basesyntax.strategy.impl.OperationStategyImpl;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TransationProcessorImplTest {
+    private Storage storage;
+    private FruitService fruitService;
+    private SupplyOperationHandler supplyOperationHandler;
+    private ReturnOperationHandler returnOperationHandler;
+    private BalanceOperationHandler balanceOperationHandler;
+    private PurchaseOperationHandler purchaseOperationHandler;
+    private OperationStategy operationStategy;

```

Comment:
all of that is not used in the actual test, and can be removed, and handlers can be a local method variables in `setUp()`